package.json

```diff
@@ -12,18 +12,17 @@
   },
   "dependencies": {
     "bindings": "^1.3.0",
-    "buffer-alloc": "^1.1.0",
-    "debug": "^3.0.1",
-    "nan": "^2.6.2",
-    "readable-stream": "^2.3.3"
+    "debug": "^4.0.0",
+    "readable-stream": "^3.0.0"
   },
   "devDependencies": {
-    "@types/node": "^10.11.4",
-    "mocha": "^3.5.0",
-    "standard": "^10.0.3"
+    "@types/node": "^12.12.14",
+    "buffer-alloc": "^1.1.0",
+    "mocha": "^6.2.2",
+    "standard": "^14.3.1"
   },
   "engines": {
-    "node": ">4"
+    "node": ">=8.6"
```

Comment:
Out of curiosity, why `8.6` instead of just `8`?

Reply:
Some things in the N-API, to be honest, I don't exactly remember which function it was 🙈

Reply:
It's:

- `napi_deferred`
- `napi_throw_error` with a third parameter
- `napi_create_string_*` using `NAPI_AUTO_LENGTH`